docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
 
 ### Visualization
 
-- Support hardware anti-aliasing.
-- Thematic display of point clouds and terrain.
-- Support for compressed texture formats.
-- Large-scale scene animations (e.g., pedestrian simulations, weather animations, disaster simulations).
-- Support atmospheric rendering effects.
+- [X] Support hardware anti-aliasing.
+- [X] Thematic display of point clouds and terrain.
+- [ ] Support for compressed texture formats.
+- [X] Particle effects
+- [X] Screen-space effects
+- [ ] Large-scale scene animations (e.g., pedestrian simulations, weather animations, disaster simulations).
+- [ ] Support atmospheric rendering effects.
 
 ### Extensions
 
-- Enhance publishing workflow to Bentley's Extension Service.
-- Create examples and templates for building and publishing Extensions.
-- Support for better testing in the Extension's Hosting App, i.e. iTwin Viewer, iTwin Design Review, PlantSight, etc.
+- [ ] Enhance publishing workflow to Bentley's Extension Service.
+- [ ] Create examples and templates for building and publishing Extensions.
+- [ ] Support for better testing in the Extension's Hosting App, i.e. iTwin Viewer, iTwin Design Review, PlantSight, etc.
 
 ### GIS Display
 
-- Expand base map support to include additional sources (AzureMaps, MapBox etc.) or solid color. Transparency of base map can be controlled independently of layers.
-- Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
+- [X] Expand base map support to include additional sources (AzureMaps, MapBox etc.) or solid color. Transparency of base map can be controlled independently of layers.
+- [X] Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
 
 ### Microsoft Teams integration
 
-- Provide api for hosting iTwin.js applications directly within Microsoft Teams.
+- [ ] Provide api for hosting iTwin.js applications directly within Microsoft Teams.
```

Comment:
@aruniverse were you involved in integrating iTwin.js into Teams a while ago?

Reply:
I was not, it was a team running in parallel with Design Review. @Carl-Hinkle or @jason-crow might remember who and can comment on the status

Reply:
I think that was Rakesh, I'm not sure what his GitHub id is.

Reply:
it was Rakesh Kancharla and Martynas Mazeika